thrust/thrust/type_traits/is_trivially_relocatable.h

```diff
@@ -40,8 +40,7 @@
 #include <cuda/std/__fwd/pair.h>
 #include <cuda/std/__fwd/tuple.h>
 #include <cuda/std/__type_traits/conjunction.h>
-
-#include <type_traits>
+#include <cuda/std/type_traits>
```

Comment:
Q: I assume we included only one internal type trait header (the line above) for a reason? Do we want everything here?

Reply:
I simply blindly replaced `#include <type_traits>` with `#include <cuda/std/type_traits>` to ensure this is compilable by NVRTC. Hadn't dug into the actual usage of it.

My guess is we do need this for things like `::cuda::std::is_trivially_copyable`